src/App.js

```diff
@@ -1,9 +1,106 @@
+import { useState } from "react";
+import './style.css'; //이번 과제는 컴포넌트 분리가 어려워서 css를 사용했습니다 ㅠㅠ
+import './background.js';
+import Clock from './clock.js';
 
 function App() {
+  const [toDo, inputToDo] = useState("");
+  const [toDos, setToDos] = useState([]);
```

Comment:
`toDo`와 `toDos`가 헷갈릴 수 있을 것 같아요
```suggestion
  const [toDo, inputToDo] = useState("");
  const [toDoList, setToDoList] = useState([]);
```
이런 식으로 작성하는 게 더 좋아 보입니다

Reply:
훨씬 깔끔하네요! 감사합니당